### PR TITLE
LRDOCS-7939 Semantic Versioning Wordsmithed

### PR DESCRIPTION
**Pull request by sez11a**
Wordsmithed Jim's port. @jhinkey

https://issues.liferay.com/browse/LRDOCS-7939

### CODE REVIEW

File: docs/dxp/7.x/en/liferay-internals/fundamentals/semantic-versioning.md [hunk 1]
```diff
@@ -1,3 +1,87 @@
...
+
+Tracking a dependency's exact version is safer, but less flexible. You could be limited to a specific DXP version or locked into APIs that exist only in that specific version. Your module, however, is much easier to test and has less chance for unexpected failures.
+
+```note::
+   When specifying package versions in your ``bnd.bnd`` file, exact versions are typically specified like this: ``version="1.1.2"``. However, this syntax is technically a range; it is interpreted as [1.1.2, &#8734;). Therefore, if a higher version of the package is available, it's used instead of the version you specified. For these cases, it may be better to specify a version range for compatible versions that have been tested. If you want to specify a true exact match, the syntax is like this: ``[1.1.2]``. See the `Version Range <https://osgi.org/specification/osgi.core/7.0.0/framework.module.html#i3189032>`_ section in the OSGi specifications for more info.
```
Comment (jrhoun) [inline docs/dxp/7.x/en/liferay-internals/fundamentals/semantic-versioning.md:74]: > it is interpreted as [1.1.2, &#8734;).

Should that be an infinity symbol, or is this text correct?

**Comment by jrhoun** [comment]
Merged. Thanks. Left the text from my [comment](https://github.com/jrhoun/liferay-learn/pull/567#discussion_r478735505) as is -- but please send another pull if that does need to be corrected.

File: docs/dxp/7.x/en/liferay-internals/fundamentals/semantic-versioning.md (hunk 1 above)
Comment (jhinkey) [reply docs/dxp/7.x/en/liferay-internals/fundamentals/semantic-versioning.md:74]: @jrhoun - Yeah, it is supposed to render as an infinity symbol, but it's not rendering when I build the site. I'm investigating how to render it in the RST syntax.

**Comment by jhinkey** [comment]
Corrected in https://github.com/jrhoun/liferay-learn/pull/600 Thanks.

**Pull request closed without merging**